end-to-end-tests/cypress/e2e/error-messages/advisory-board-date.cy.js

```diff
@@ -25,8 +25,9 @@ describe('Tests to check advisory board date error messages', { tags: '@dev'}, (
         cy.clickDataTest("ab-date");
         cy.fillInDate(Cypress.dayjs().subtract(1,'M'))
         cy.get('.govuk-button').click();
-        cy.get('.govuk-error-summary__body > .govuk-list > li > a').should('have.text', 'You must enter a future date').should('be.visible');        
-        cy.get('#AdvisoryBoardViewModel\\.AdvisoryBoardDate\\.Date-error').should('have.text', 'Error:You must enter a future date').should('be.visible');
+        cy.url().then(href => {
```

Comment:
it could be cy.url().should('include', '/transfer-dates')
however if this works fine by me :)